auto_py_to_exe/web/index.html

```diff
@@ -52,7 +52,7 @@
                         <img src="https://github.githubassets.com/favicons/favicon.png" alt="GitHub favicon" />
                     </a>
                     <a href="https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/?utm_source=auto_py_to_exe&utm_medium=application_link&utm_campaign=auto_py_to_exe_help&utm_content=top" target="_blank">
-                        Help Post
```

Comment:
Could you remove the changes from this file and create a new PR with these changes targeting `feature/153-add-chinese-translations`? I have made a new file [i18n.js](https://github.com/brentvollebregt/auto-py-to-exe/blob/feature/153-add-chinese-translations/auto_py_to_exe/web/js/i18n.js) which can hold all the translations.